src/lib/messages/mission.proto

```diff
@@ -178,6 +178,8 @@ message MissionPlan
         ];
     }
     optional Speeds speeds = 5;
+    optional uint32 fragment_index = 6 [(dccl.field) = { min: 0 max: 255 }];
```

Comment:
Isn't this hardcoded to max: 2 at this point (for three fragments)? Why 255 then?

Reply:
Good point